docker/bootstrap/Dockerfile.percona57

```diff
@@ -5,16 +5,15 @@ FROM --platform=linux/amd64 "${image}"
 
 # Install Percona 5.7
 RUN for i in $(seq 1 10); do apt-key adv --no-tty --keyserver keyserver.ubuntu.com --recv-keys 9334A25F8507EFA5 && break; done && \
-    add-apt-repository 'deb http://repo.percona.com/apt buster main' && \
+    add-apt-repository 'deb http://repo.percona.com/apt bullseye main' && \
     { \
         echo debconf debconf/frontend select Noninteractive; \
         echo percona-server-server-5.7 percona-server-server/root_password password 'unused'; \
         echo percona-server-server-5.7 percona-server-server/root_password_again password 'unused'; \
     } | debconf-set-selections && \
     apt-get update && \
-    apt-get install -y --no-install-recommends \
-        percona-server-server-5.7 \
-	libperconaserverclient20-dev percona-xtrabackup-24 && \
+    apt-get install -y --no-install-recommends percona-server-server-5.7 && \
+    apt-get install -y --no-install-recommends libperconaserverclient20-dev percona-xtrabackup-24 && \
```

Comment:
Could we not merge these commands?

Reply:
That led to unexpected behaviors where `percona-xtrabackup-24` was going through an apt-get setup phase was left pending for hours. Splitting the command in two solved the issue.